src/three-components/TextureUtils.ts

```diff
@@ -260,6 +254,115 @@ export default class TextureUtils extends EventDispatcher {
     }
   }
 
+  gaussianBlur(
+      cubeTarget: WebGLRenderTargetCube,
+      standardDeviationRadians: number = 0.04) {
+    const blurScene = new Scene();
+
+    const geometry = new BoxBufferGeometry();
+    geometry.removeAttribute('uv');
+
+    const cubeResolution = cubeTarget.width;
+    const standardDeviations = 3;
+    const n = Math.ceil(
+        standardDeviations * standardDeviationRadians * cubeResolution * 2 /
+        Math.PI);
+    const norm = standardDeviations / ((n - 1) * Math.sqrt(2 * Math.PI));
+    let weights = [];
+    for (let i = 0; i < n; ++i) {
+      const x = standardDeviations * i / (n - 1);
+      weights.push(norm * Math.exp(-x * x / 2));
+    }
+
+    const blurMaterial = new ShaderMaterial({
+      uniforms: {
+        tCube: {value: null},
+        latitudinal: {value: false},
+        weights: {value: weights},
+        dTheta: {value: standardDeviationRadians * standardDeviations / (n - 1)}
+      },
+      vertexShader: `
+        varying vec3 vWorldDirection;
+        #include <common>
+        void main() {
+          vWorldDirection = transformDirection( position, modelMatrix );
+          #include <begin_vertex>
+          #include <project_vertex>
+          gl_Position.z = gl_Position.w;
+        }
+      `,
+      fragmentShader: [
+        'const float n = ' + n + '.0;',
+        'uniform float weights[' + n + '];',
```

Comment:
A couple of comments...

JavaScript numbers have a [`toFixed`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed) method that would probably be appropriate here.

You can also interpolate values with the template string (the one that uses the ` character as its boundary) that follows. For example:

```typescript
const myProgram = `
const float n = ${n.toFixed(1)};
uniform float weights[${n}];

// rest of the program follows ...
`;
```

Template strings are also multi-line friendly, so this will save you from the cumbersome and less legible `.join('\n')` on an array of strings.